### ISSUE TITLE
Problem downloading vimeopro video

### ISSUE THREAD

**Issue by rcamargo18** [rg3/youtube-dl#16340]
Hi there;

I got a problem when I tried to download a video from vimeopro.com. Debug screen shows like below. Please help me. Thank you.

youtube-dl https://player.vimeo.com/video/88810243 --referer https://vimeopro.com/projectical/curso-de-gestion-avanzada-de-riesgos-en-proyectos/page/1 

youtube-dl "https://player.vimeo.com/video/88810243" --referer "https://vimeopro.com/projectical/curso-de-gestion-avanzada-de-riesgos-en-proyectos/page/1" 
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://player.vimeo.com/video/88810243', '--referer', 'https://vimeopro.com/projectical/curso-de-gestion-avanzada-de-riesgos-en-proyectos/page/1', '--video-password', 'PRIVATE', '--verbose']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2018.05.01
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.16299
[debug] exe versions: ffmpeg N-73361-g03b2b40, ffprobe N-73361-g03b2b40
[debug] Proxy map: {}
[vimeo] 88810243: Downloading webpage
ERROR: Cannot download embed-only video without embedding URL. Please call youtube-dl with the URL of the page that embeds this video.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 519, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\YoutubeDL.py", line 2203, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\vimeo.py", line 480, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 539, in _download_webpage_handle
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 528, in _request_webpage
youtube_dl.utils.ExtractorError: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 519, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\YoutubeDL.py", line 2203, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\vimeo.py", line 480, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 539, in _download_webpage_handle
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 528, in _request_webpage
youtube_dl.utils.ExtractorError: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\YoutubeDL.py", line 789, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\common.py", line 440, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpulhs0gaq\build\youtube_dl\extractor\vimeo.py", line 495, in _real_extract
youtube_dl.utils.ExtractorError: Cannot download embed-only video without embedding URL. Please call youtube-dl with the URL of the page that embeds this video.

**Comment by kode54** [comment]
I'm having a different problem with a different vimeopro video:

```
$ youtube-dl --verbose https://vimeopro.com/andyet/andyet-conf/video/175370577
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://vimeopro.com/andyet/andyet-conf/video/175370577']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.06.19
[debug] Python version 3.6.5 (CPython) - Darwin-17.6.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 4.0.1, ffprobe 4.0.1
[debug] Proxy map: {}
[vimeo] 175370577: Downloading webpage
[vimeo] 175370577: Extracting information
[vimeo] 175370577: Downloading webpage for description
[vimeo] 175370577: Downloading JSON metadata
WARNING: Unable to download JSON metadata: HTTP Error 403: Forbidden
ERROR: An extractor error has occurred. (caused by KeyError('title',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 501, in extract
    ie_result = self._real_extract(url)
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 643, in _real_extract
    info_dict_config = self._parse_config(config, video_id)
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 110, in _parse_config
    video_title = video_data['title']
KeyError: 'title'
Traceback (most recent call last):
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 501, in extract
    ie_result = self._real_extract(url)
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 643, in _real_extract
    info_dict_config = self._parse_config(config, video_id)
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/vimeo.py", line 110, in _parse_config
    video_title = video_data['title']
KeyError: 'title'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 792, in extract_info
    ie_result = ie.extract(url)
  File "/opt/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 514, in extract
    raise ExtractorError('An extractor error has occurred.', cause=e)
youtube_dl.utils.ExtractorError: An extractor error has occurred. (caused by KeyError('title',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by remitamine** [comment]
> https://vimeopro.com/projectical/curso-de-gestion-avanzada-de-riesgos-en-proyectos/page/1
can't be checked as it require a password
> https://vimeopro.com/andyet/andyet-conf/video/175370577
should be fixed with 06242d44fe261999e2424d9ecb00f20ff30ccb9b.
